veba-bom.json

```diff
@@ -87,47 +76,71 @@
             }
         ]
     },
-    "knative": {
-        "gitRepoTag": "v0.20.0",
-        "containers": [{
-                "name": "gcr.io/knative-releases/github.com/projectcontour/contour/cmd/contour@sha256",
-                "version": "3e88236949d49b0933318ef077741e9bf6f40ae84e8b79d7c229e1aaff53220f"
+    "knative-serving": {
+        "gitRepoTag": "v0.26.0",
+            "containers": [{
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/queue@sha256",
+                "version": "5af97fa5e19c6fd1e04642e7cf6eb01e6982b4cfee19d8fd16649423277c2eb9"
             },
             {
-                "name": "envoyproxy/envoy",
-                "version": "v1.16.0"
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/activator@sha256",
+                "version": "00aeb9267dc6445bbb11cb90636e9cde44404c6303d55b81aa074381b4989eef"
             },
             {
-                "name": "gcr.io/knative-releases/knative.dev/net-contour/cmd/controller@sha256",
-                "version": "5654ef5be4f506909bba6cbb59e7049d6c4bfc866066f2546bcabd2ff547307f"
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/autoscaler@sha256",
+                "version": "ca3bad368a2ac40f33ad9e47c1075cc2b833301b4bc772fb84c51f52cc1c0a35"
             },
             {
-                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/activator@sha256",
-                "version": "cbeb1cdcdde8b3b44bc5eb1c5a07512a36f4bd7fbee5e9677e8789ee2459b6ed"
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/controller@sha256",
+                "version": "50fcbbc79b1078991280bf423e590c8904882dc8750c7f7d61bc06d944a052f2"
             },
             {
-                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/autoscaler@sha256",
-                "version": "73d4d0503544ea930caa4fba07b701147339131de31a6aae8c52d6bb568f3c81"
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/domain-mapping@sha256",
+                "version": "3b7da888fafca8cc5ba5e2aa62d6f97751d50890ed9d0b01aabce66a7d26351e"
             },
             {
-                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/controller@sha256",
-                "version": "3054c326df9de07a2a1ce34a2515339201706995a16f8efc8228295a6b94fbdd"
+                "name": "gcr.io/knative-releases/knative.dev/serving/cmd/domain-mapping-webhook@sha256",
+                "version": "a6529f0625483c81741c92895e4d54be8a103ecc5801e7c3aa049d3b3ea7cc90"
             },
             {
                 "name": "gcr.io/knative-releases/knative.dev/serving/cmd/webhook@sha256",
-                "version": "07894526a4b49c61c5333e4f48190e830ce5124f0c18e1042cc15fcb594bfc37"
+                "version": "382a1d64ea0686da2b973c95be96fcae29ac25b256b90f5735b1479a93d19c7a"
+            }
+        ]
+    },
+    "knative-eventing": {
+        "gitRepoTag": "v0.25.1",
```

Comment:
nit: align this with serving?

Reply:
IIRC, there wasn't v0.26 and latest (at the time of this PR) was v0.25.1